### PR TITLE
Added issue 3268 to the whats_new.rst

### PR DESCRIPTION
**Pull request by gamazeps**
Also removed some useless (in my opinion) code in `mne/test/test_utils.py`

### CODE REVIEW

File: doc/whats_new.rst [hunk 1]
```diff
@@ -77,6 +77,12 @@ API
 
     - Add ``zorder`` parameter to :func:`mne.Evoked.plot` and derived functions to sort allow sorting channels by e.g. standard deviation, by `Jona Sassenhagen`_
 
+    - The ``baseline`` parameter of :func:`mne.Epochs.apply_baseline` is set by default (None, 0), by `Felix Raimundo`_
+
+    - Adds :func:`mne.Evoked.apply_baseline` to be consistent with :func:`mne.Epochs.apply_baseline`, by `Felix Raimundo`_
+
+    - Deprecated the `baseline` parameter in :func:`mne.Evoked.__init__`, by `Felix Raimundo`_
```
Comment (agramfort) [inline doc/whats_new.rst]: @gamazeps you need to put the link to your homepage or github page at the bottom ortherwise the RST link will not work.

Comment (mmagnuski) [inline doc/whats_new.rst]: just BTW - my link doesn't work in what's new - what did I do wrong? Is the letter 'ł' to blame? :)

Comment (mmagnuski) [inline doc/whats_new.rst]: I mean - it works in the .rst but not on the mne dev webpage what's new

Comment (Eric89GXL) [inline doc/whats_new.rst]: `__init__` doesn't get document itself, but rather as the class constructor so instead do:
```
... in the constructor for :class:`mne.Evoked`, by ...
```

Comment (gamazeps) [inline doc/whats_new.rst]: No idea, I guess i did something wrong ?

Comment (agramfort) [inline doc/whats_new.rst]: I fix in master

**Pull request merged**